debian/control

```diff
@@ -0,0 +1,22 @@
+Source: libpam-policycache
+Priority: extra
+Maintainer: Nikki VonHollen <vonhollen@google.com>
+Build-Depends: debhelper (>= 8.0.0), autotools-dev, libglib2.0-dev,
+ libpam0g-dev, automake, libtool, autogen, libscrypt-dev
+Standards-Version: 3.9.2
+Section: libs
+Homepage: https://github.com/google/libpam-policycache
+
+Package: libpam-policycache
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, libscrypt0
```

Comment:
I get this:

> $ cat debian/libpam-policycache.substvars 
> shlibs:Depends=libc6 (>= 2.4), libglib2.0-0 (>= 2.31.8), libpam0g (>= 1.1.0), libscrypt0 (>= 1)

So for me it just works. Now, to be fair, I also get a bunch of warnings like this:

> dpkg-shlibdeps: warning: tried to merge the same object (libglib-2.0.so.0) twice in a symfile